packages/custom-types/message-types.d.ts

```diff
@@ -0,0 +1,12 @@
+import { TemplateBody } from "./template";
+
+export interface TemplateBodysByKey { [property: string]: TemplateBody }
+
+interface MessageTypes {
```

Comment:
@lilitkarapetyan - we have two elements with very similar names here.   Can you explain the difference between them please?

Reply:
TemplateBody[] - it's array of templates. if you want get some template from there you need to do loop find
TemplateBodysByKey - it's object of templates. if you want to get some template from there you need to do TemplateBodysByKey[yourTemplateId] and you will get it without any loop findes

Reply:
I should have been more clear 🤣 

I was talking about `loadingDb` and `isLoading` in the next two lines.